content/en/docs/File Format/implementationstatus.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Implementation status"
+linkTitle: "Implementation status"
+weight: 8
+---
+### Physical types
+
+| Data type                                 | C++   | Java   | Go    | Rust  |
+| ----------------------------------------- | ----- | ------ | ----- | ----- |
+| BOOLEAN                                   |       |        |       |       |
+| INT32                                     |       |        |       |       |
+| INT64                                     |       |        |       |       |
+| INT96                                     |       |        |       |       |
```

Comment:
Should it be noted that INT96 is deprecated?

Reply:
Alternatively, INT96 can stay as is in the main column as it's a part of the spec and individual implementations can indicate their status?

Reply:
I would rather note "INT96 (deprecated)". It is deprecated in the spec, not only in individual implementations. People not experts in Parquet and looking at this matrix should not get the impression that implementations without INT96 are somehow lagging behind the others.

Reply:
Unfortunately, while deprecated some writers like [Spark](https://spark.apache.org/docs/latest/sql-data-sources-parquet.html#configuration) still output by default I believe :(  I agree it should stay here and maybe we can have a subscript detailing this.